packages/docmanager-extension/src/index.ts

```diff
@@ -539,7 +540,7 @@ function addCommands(
   commands.addCommand(CommandIDs.restoreCheckpoint, {
     label: () =>
       trans.__(
-        'Revert %1 to Checkpoint',
+        'Revert %1 to Checkpoint...',
```

Comment:
Use Unicode ellipsis  character (…) see #10000:
 
```suggestion
        'Revert %1 to Checkpoint…',
```